crates/l2/storage/src/store.rs

```diff
@@ -51,12 +55,16 @@ impl Store {
             EngineType::RedB => Self {
                 engine: Arc::new(RedBStoreRollup::new()?),
             },
+            #[cfg(feature = "sql")]
+            EngineType::SQL => Self {
+                engine: Arc::new(SQLStore::new(_path).await?),
```

Comment:
Is there no sync variant for this? An async new seems weird at least.

Reply:
Can we research if we can instance the SQL DB in a sync function? I think it's worth researching since having an async "new" function is weird and can be poor to use in a non-async context. Maybe we could spawn a local tokio executor, call the builder, and get the result.